x5.md

```diff
@@ -117,215 +79,45 @@ Status generates a deterministic 3-word random pseudonym from the whisper chat p
-
-#### Inattentive User Resistant (YES)
-> Users do not need to carefully inspect information (e.g., key fingerprints) to achieve security.
-
-- Currently and in general, yes.
-- It is worth mentioning that we also use three random names as a shorthand mapping for public keys. This is useful to casually establish provenance in a public chat. Additionally we have display names that are user set (and visible to someone who is added as a contact), which could lead to impersonation attacks.
-- Note that the possible amount of 3-word pseudonyms is drastically smaller than the possible amount of public keys, so there is potential for collisions, and therefor impersonation attacks here as well.
-
-### --- Adoption Properties
-
-#### Multiple Key Support (NO)
-> Users should not have to invest additional effort if they or their conversation partners use multiple public keys, making the use of multiple devices with separate keys transparent. While it is always possible to share one key on all devices and synchronize the key between them, this can lead to usability problems.
-
-- In the future, Universal logins will help with this, as it manages multiple keys and their associated device and risk, and also enables multi-factor authentication for various features. 
-- Furthermore, HD wallets could add some multi-key functionality (e.g. different keypairs on each device), but it would probably be best to stick with universal logins.
-
-comments: 
-- potential for self-signed attestations to universal blacklist for compromised keys. (sparse merkle tree for lookup)
-
-#### No Service Provider Required (YES)
-> Trust establishment does not require additional infrastructure (e.g., key servers).
- 
-- Not required for trust establishment
-
-#### No Auditing Required (YES)
-> The approach does not require auditors to verify correct behavior of infrastructure operators.
-
-- Not required for trust establishment
-
-#### No Name Squatting (PARTIAL)
-> Users can choose their names and can be prevented from reserving a large number of popular names
-
-- In general, yes.
-- Public keys have a big space. For three-random words it is possible to generate arbitrary combinations with decent computational resources, but this doesn't amount to "reserving" it.
-- For ENS, no. But for each name, a stake of 10 SNT is required, as well as the creation of a new identity (unless they interact directly with the ENS registration contract directly and not through the app)
-
-#### Asynchronous (YES)
-> Trust establishment can occur asynchronously without all conversation participants online.
+possible connections
+- client - client (not really ever, this is facilitated through all other connections)
+    - personal chat
+        - ratcheted with X3DH
+    - private group chat
+        - pairwise ratcheted with X3DH
+    - public chat
+- client - mailserver (statusd + ???)
```

Comment:
What's the action to get clarity on these unknowns? Same as rendezvous info below